units/L_Ghast.cfg

```diff
@@ -2,41 +2,20 @@
 
 [unit_type]
     id=L Ghast
+    [base_unit]
+        id=Ghast
+    [/base_unit]
     name= _ "Cursed Ghast"
-    race=undead
-    {TRAIT_FEARLESS_MUSTHAVE}
-    image="units/ghast2/ghast2.png"
-    profile="portraits/undead/ghoul.png"
-    {MAGENTA_IS_THE_TEAM_COLOR}
-    hitpoints=65
-    movement_type=gruefoot
-    movement=5
-    experience=150
-    level=3
-    alignment=chaotic
-    advances_to=null
-    {AMLA_DEFAULT}
-    cost=41
-    usage=fighter
     description= _ "This ghast is put under a curse by Ancient Lich. As a result, its poison can work against even undead units."+{SPECIAL_NOTES}+{SPECIAL_NOTES_POISON}+{SPECIAL_NOTES_FEEDING}+{SPECIAL_NOTES_REGENERATES}+{SPECIAL_NOTES_ARCANE}
-    die_sound=ghoul-hit.wav
     [abilities]
         {ABILITY_FEEDING}
         {ABILITY_REGENERATES}
     [/abilities]
-    {DEFENSE_ANIM "units/ghast2/ghast2-defend-2.png" "units/ghast2/ghast2-defend-1.png" ghoul-hit.wav }
+
+    # because of [base_unit] we need this, otherwise we change the first attack
     [attack]
-        name=bite
-        description={STR_BITE}
-        icon=attacks/fangs.png
-        type=blade
-        range=melee
-        damage=10
-        number=3
-        [specials]
-            {WEAPON_SPECIAL_POISONG}
-        [/specials]
```

Comment:
What I meant is that here https://github.com/konecnyjakub/For_Power/blob/54c43c3ebd498ac7c3125fb404ed708fc630407d/units/humans/cult.cfg#L13 the first attack inherits weapon special from the base unit.